contrib/aws/cloudformation-ec2-efs.yaml

```diff
@@ -159,7 +159,7 @@ Resources:
             mount -a -t efs defaults
 
             # Create/update the config & password files.
-            mkdir "$BASE/conf"
+            mkdir "$BASE/conf" -p
```

Comment:
Good catch! Not sure how I missed this...

Should be `mkdir -p "$BASE/conf"`. But I think this change would be better:

```bash
[[ -d "$BASE/conf" ]] || mkdir "$BASE/conf"
```

The goal is to attempt to create the directory only if it doesn't exist.
`mkdir -p` tries to create all parent directories if they don't exist. My suggestion preserves the original intent a little more clearly.

Reply:
exactly! I've change to use test:)